### DISCUSSION TITLE
[BUG] can't use Background remover example with back camera of iOS in Safari .

### DISCUSSION THREAD

**Discussion by predictcare** [ant-media/Ant-Media-Server#5707] [Q&A]
Hi, we just tried to make the custom_video_source_demo.html working with the back camera of mobile devices.

We just took the example html and added the line 245 :
function startPublishing() {
		webRTCAdaptor.bandwidth = maxVideoBitrateKbps;
		_**webRTCAdaptor.switchVideoCameraFacingMode(streamId,"environment");**_
		streamId = streamIdBox.value;
		webRTCAdaptor.publish(streamId, token, subscriberId, subscriberCode, streamName, mainTrack);
	}

It won't work, we have either black screen or face front camera.

Can you please fix it ?

Thanks !

Victor

**Comment by zerostack0** [comment]
Hi Victor,

Thanks for reaching out to us about the bug. I created an issue for that and we will schedule it soon.
Let me explain how we work. We have weekly sprints and schedule the issues from GitHub directly.
If you notice a bug and have reproduced the scenario, you can create an issue directly,  instead of QA.

https://github.com/ant-media/Ant-Media-Server/issues/5713
